hivanreyes/oe-webapp
# Feat - 497 - Mapbox Home

#### What does this PR do?
Implement mapboxgl.

#### Where should the reviewer start?
DynamicMap.jsx is where the magic happens: `components/shared/maps/ExpeditionsMap/DynamicMap.jsx`

#### How should this be manually tested?
install all the new dependencies: `yarn add`. And open the app. Make sure your backend server is running.

#### What are the relevant tickets?
[NGPOE-497](https://fng-jira.fox.com/browse/NGPOE-497)

#### Checklist
